airbyte-cdk/java/airbyte-cdk/core/src/main/kotlin/io/airbyte/cdk/integrations/util/ConnectorExceptionUtil.kt

```diff
@@ -30,13 +31,18 @@ object ConnectorExceptionUtil {
     fun isConfigError(e: Throwable?): Boolean {
         return isConfigErrorException(e) ||
             isConnectionError(e) ||
-            isRecoveryConnectionException(e) ||
             isUnknownColumnInFieldListException(e)
     }
 
+    fun isTransientError(e: Throwable?): Boolean {
+        return isTransientErrorException(e) || isRecoveryConnectionException(e)
+    }
+
     fun getDisplayMessage(e: Throwable?): String? {
         return if (e is ConfigErrorException) {
             e.displayMessage
+        } else if (e is TransientErrorException) {
+            e.message
```

Comment:
shouldn't this be e.displayMessage?

Reply:
Ah no - we're using the parent method which is just message